### PR TITLE
Fixes cucumbers tests in porta

### PR DESCRIPTION
**Pull request by josemigallas**

### CODE REVIEW

**Review by hallelujah — APPROVED** [review]
That fixed it

File: Dockerfile [hunk 1]
```diff
@@ -96,6 +97,8 @@ VOLUME [ "/opt/system/tmp/cache/", \
          "/home/ruby/.luarocks" \
        ]
 
+RUN dbus-uuidgen | sudo tee -a /etc/machine-id
```
Comment (gsaslis) [inline Dockerfile]: every new command in the dockerfile adds a new layer in the docker image. 
do you think we could move this to one of the other `RUN` blocks?

**Comment by gsaslis** [comment]
i still didn't understand what it was that we changed that brought this requirement upon us. 

i mean, cucumber tests were passing for quite some time without this... 

have we spotted if there was any change in particular that triggered this? (e.g. library version change, some code in our test suite, something we do differently in the circleci setup, etc...)

**Comment by gsaslis** [comment]
@josemigallas can you rebase on master, so it picks up the circleci config ?
only added that this morning.. 

Thanks!

**Pull request merged**